### PR TITLE
Add Docs to add_external_moment_function

### PR DESCRIPTION
**Pull request by joaofoes**

### CODE REVIEW

File: docs/Components/BaseComponent.md [hunk 1]
```diff
@@ -98,6 +98,18 @@ base_component.add_external_force_function('force1', force1)
 print(base_component.external_forces)
 >>> {'force1': <function force1 at 0x0000025EBCAA8D30>}
 ```
+## *add_external_moment_function*:
+This method is responsible for appending an external moment function to the component and it should return the magnitude of the moment only.
```
Comment (rafaellehmkuhl) [inline docs/Components/BaseComponent.md:102]: You got it.
Just to make it clearer:

This method is responsible for appending an external moment function to the component. The moment function should return a float representing the magnitude of the moment.

File: docs/Components/BaseComponent.md [hunk 2]
```diff
@@ -98,6 +98,18 @@ base_component.add_external_force_function('force1', force1)
...
+```python
+def moment1():
+    moment1 = 13
+    return moment1
+
+base_component.add_external_moment_function('moment1', moment1)
+
+print(base_component.external_moments)
+>>> {'moment1': 13}
```
Comment (rafaellehmkuhl) [inline docs/Components/BaseComponent.md:111]: Isn't it returning something like ```{'moment1': <function moment1 at 0x7ff87d8891f0>}```?
When we print the function instance, it doesn't evaluate the function (and so returning the value 13).

**Review by rafaellehmkuhl — CHANGES_REQUESTED** [review]

File: docs/Components/BaseComponent.md (hunk 2 above)
Comment (joaofoes) [reply docs/Components/BaseComponent.md:111]: It problably is, but as i'm having the issue with the pytest, i'm not able to run the test and see the exact return of the print without having to manipulate the code to show it. Sorry.

Comment (rafaellehmkuhl) [reply docs/Components/BaseComponent.md:111]: No problem. I will try to help you with the pytest problem today.

File: docs/Components/BaseComponent.md [hunk 3]
```diff
@@ -99,13 +99,13 @@ print(base_component.external_forces)
 >>> {'force1': <function force1 at 0x0000025EBCAA8D30>}
 ```
 ## *add_external_moment_function*:
-This method is responsible for appending an external moment function to the component and it should return the magnitude of the moment only.
+This method is responsible for appending an external moment function to the component and it should return a float representing the magnitude of the moment.
```
Comment (rafaellehmkuhl) [inline docs/Components/BaseComponent.md]: I think we could yet improve this statement, separating in two phrases:

"This method is responsible for appending an external moment function to the component. The appended function should return a float representing the magnitude of the moment."

**Review by rafaellehmkuhl — CHANGES_REQUESTED** [review]

**Review by rafaellehmkuhl — APPROVED** [review]

**Review by CassianoMontibeller — APPROVED** [review]

**Pull request merged**